### ISSUE TITLE
not seeing private events on home page

### ISSUE THREAD

**Issue by JChipault** [USGS-WiM/whispersservices_django#362]
A NWHC partner manager (Nathan Kudla) can see events marked as private on his dashboard "Events" tab but not on the home page. He should have permissions to see private events in his organization (NWHC) on the home page, right?

**Comment by aaronstephenson** [comment]
This was directly related to #364 
Fixed in 1.5.0 226ccfc

**Comment by JChipault** [comment]
This one is not quite fixed... user account nwhcPartnerManager can see events marked as private on the "Your Events" tab
![image](https://user-images.githubusercontent.com/49287076/79758785-75935b80-82e3-11ea-8f08-26daaf29badc.png)

seems to only get private events showing on "Events" tab AFTER refreshing the search. So if start on the home page, then go to "Your Events" in dashboard, I see private event (shown above), then I flip to "Events" and I do NOT see private event (even though it's within search criteria)
![image](https://user-images.githubusercontent.com/49287076/79759339-2c8fd700-82e4-11ea-980d-7d9e300ad807.png)

And then within "Events" tab I click search and then submit search (not changing any criteria) and when the page refreshes I get this:
![image](https://user-images.githubusercontent.com/49287076/79759482-5943ee80-82e4-11ea-8c20-f09e5709c626.png)

I can't see the event on the home page even if I refresh the search
![image](https://user-images.githubusercontent.com/49287076/79759573-77115380-82e4-11ea-9493-66adf8d87038.png)

**Comment by BlakeDraper** [comment]
@JChipault this is curious. I've logged in to the test site with the nwhcPartnerManager account and done the things you have described 4 times, and I see all the private events every time in every place, with no refresh needed. See images below. Not sure why you are having a different result. Are you on v2.14.4 in the test environment and logged out of the API interface?

@lprivette would you test this too? I'll send you the password for that account privately. 

![image](https://user-images.githubusercontent.com/6978380/79778533-784f7a00-82fe-11ea-9bd0-e4c839b45ed4.png)

![image](https://user-images.githubusercontent.com/6978380/79778588-95844880-82fe-11ea-8258-b5ee8f924ebb.png)

![image](https://user-images.githubusercontent.com/6978380/79778667-bea4d900-82fe-11ea-95c4-9bc975a2aa19.png)

**Comment by lprivette** [comment]
Yep, testing now

**Comment by BlakeDraper** [comment]
@lprivette somehow this appears to be a Firefox issue. On Firefox I do see the reported behavior. Please try both Chrome and FF and see if you can confirm?

**Comment by JChipault** [comment]
I get the same errant behavior in chrome. 

I do have 12.4.4 and am logged out of the API. 

This originally hit our radar when user Nathan Kudla (also a partner manager) noticed that his "Your Events" tab showed events that he wasn't seeing in his "Events" tab or on the home page. Not sure what browser he was using, but it's not just my computer that's having this issue.

**Comment by BlakeDraper** [comment]
@JChipault Lauren and I did some sleuthing and found out what is going on here. Its not actually a bug. My mention of Firefox turned out to be a red herring. 

It it not that the private events are not showing - its just that those three lists all display the results in a different way. I think you will find that if you sort the home page results differently or just increase the results page size, Event 170878 shows up. It was there all along, just not in the same order as you saw elsewhere. The home page results are handled in a different way from the server to support the results mapping on that page. 

As for the user dashboard Events and Your Events tabs - same deal - the results are all there, they just don't show in the same order. One small tweak I have made is to be sure both of those tabs are querying the server by start date with latest events first. Currently only the Events tab is doing that.

Even with this change (which will come with next release), you will not always seem the same results between those two tabs because they show different things. Your Events show all of the users's events, irrespective of the search.
